app/src/main/java/com/github/polypoly/app/map/MapActivity.kt

```diff
@@ -88,7 +108,11 @@ class MapActivity : ComponentActivity() {
                 ) {
                     MapView()
                     BuildingInfoUIComponent()
-                    Hud(PlayerGlobalData(false, 420), listOf(PlayerGlobalData(false, 32), PlayerGlobalData(false, 56)),16)
+                    Hud(
+                        PlayerGlobalData(false, 420),
+                        listOf(PlayerGlobalData(false, 32), PlayerGlobalData(false, 56)),
+                        16
+                    )
```

Comment:
I belief these PlayerGlobalData are placeholders until therer is a way to retrieve the real ones. If so, they should be declared separately and explicitly named as such instead of being inlined within the code as it is right now (it leaves magic constants coming from nowhere and is not very meaningful)

Reply:
Not my PR!

Reply:
Ok you simply fixed the indentation so this can be left to a later refactoring if needed as it is nothing big.